### PR TITLE
{lang} [goolf/1.7.20] Python v2.7.11

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

File: easybuild/easyconfigs/p/Python/Python-2.7.11-goolf-1.7.20.eb [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+scipyversion = '0.17.0'
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+
+# python needs bzip2 to build the bz2 package
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.8'),
+    ('libreadline', '6.2'),
```
Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-2.7.11-goolf-1.7.20.eb]: @pescobar 6.3?

File: easybuild/easyconfigs/p/Python/Python-2.7.11-goolf-1.7.20.eb [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+homepage = 'http://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
+
+numpyversion = '1.10.4'
+scipyversion = '0.17.0'
```
Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-2.7.11-goolf-1.7.20.eb]: these versions are different in the existing Python 2.7.11 easyconfigs, do we want to stick to those?

File: easybuild/easyconfigs/p/Python/Python-2.7.11-goolf-1.7.20.eb (hunk 1 above)
Comment (pescobar) [inline easybuild/easyconfigs/p/Python/Python-2.7.11-goolf-1.7.20.eb]: 6.3 doesn't work for me when using ipython. See here 

https://github.com/hpcugent/easybuild-easyconfigs/pull/2457#issuecomment-220324246
https://github.com/hpcugent/easybuild-easyconfigs/pull/2457#issuecomment-220549806

In any case if you want librealine 6.3 I will change it

File: easybuild/easyconfigs/p/Python/Python-2.7.11-goolf-1.7.20.eb (hunk 2 above)
Comment (pescobar) [inline easybuild/easyconfigs/p/Python/Python-2.7.11-goolf-1.7.20.eb]: I will choose the ones you prefer. Just tell me which versions should be added here...

File: easybuild/easyconfigs/p/Python/Python-2.7.11-goolf-1.7.20.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-2.7.11-goolf-1.7.20.eb]: https://github.com/hpcugent/easybuild-easyconfigs/issues/3085

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2021.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/ac14e057e2a3ec623bb07b3ccb2493e8 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Pull request merged**